test/rules/no-magic-numbers/default/test.ts.lint

```diff
@@ -28,3 +34,18 @@ class A {
 enum Test {
     key = 1337,
 }
+
+it('should provide a link to the landing page', () => {
```

Comment:
while it's cool to put real code here, it's also very confusing when the only thing we care about is that little index operator inside the _next_ line.

please remove the `it()` wrapper and just test the array access.